src/compressed_tensors/utils/helpers.py

```diff
@@ -119,3 +120,35 @@ def is_compressed_tensors_config(compression_config: Any) -> bool:
         return isinstance(compression_config, CompressedTensorsConfig)
     except ImportError:
         return False
+
+
+class AliasableEnum:
```

Comment:
This is not really an Enum, and seems more like an interface, should we make this an ABC?

Reply:
Good feedback, I've renamed to just `Aliasable`. Actually inheriting from the `ABC` class conflicts with the `Enum` class, but I've marked the method with `abstractmethod` to type-hint this as an abc